setup-homebrew/README.md

```diff
@@ -0,0 +1,12 @@
+# Setup Homebrew Github Action
+
+An action that sets up Homebrew environment.
```

Comment:
`An action that sets up a Homebrew environment.`

Reply:
https://github.com/Homebrew/actions/commit/a30f16b9e223fdf2b13640010b9f96e4988a74dc